16/umbraco-commerce/how-to-guides/show-discounted-prices.md

```diff
@@ -0,0 +1,117 @@
+---
+description: Learn how to show discounted prices on product pages in Umbraco Commerce.
+---
+
+# Showing Discounted Prices
+
+Imagine the following scenario: You’re preparing for a big sale and want to offer a 10% discount on all your products. With Umbraco Commerce’s powerful discount engine, discounts like these can be set up.
+
+But there's a catch. Discounts are only applied once the product is added to the cart. So how do you show the discounted price before that, right on the product page?
+
+This guide walks you through the process.
+
+{% hint style="info" %}
+The calculate adjusted prices feature was introduced in Umbraco Commerce version 16.2.0
+{% endhint %}
+
+## Prepare the Discount
+
+To start, set up an automatic discount with no rules but an Order Line Amount Reward set to apply a 10% discount to the unit price.

```

Comment:
An attempt to shorten the sentence.
Does this still make sense in the context?

```suggestion
To start, set up an automatic discount with Order Line Amount Reward set to apply a 10% discount to the unit price.

```

Reply:
@sophie-neale that works for me